### ISSUE TITLE
Unable to see team or organization repositories/images

### ISSUE THREAD

**Issue by lkupkepear** [microsoft/vscode-docker#2954]
Issue Type: <b>Bug</b>

Have a dockerhub organization and team. Add a docker user to the docker team. Once connected, open vscode with the docker extension. The login as the user that was added to the team. No organization line item displays in the registry details menu bar.

Extension version: 1.12.1
VS Code version: Code 1.56.1 (e713fe9b05fc24facbec8f34fb1017133858842b, 2021-05-06T10:08:24.325Z)
OS version: Darwin x64 19.6.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz (16 x 2400)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 3, 4|
|Memory (System)|32.00GB (0.67GB free)|
|Process Argv|-psn_0_8316910 --crash-reporter-id 4e8de503-5652-4159-bdd7-96bd91724fa4|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551cf:30291415
vspre833:30267464
pythonptprofiler:30281270
vshan820:30294714
pythondataviewer:30285071
vscus158cf:30286554
vscgsv2:30301612
vscorehovct:30302760
bridgeflightcf:30302070
vscod805cf:30301675

```

</details>

**Comment by karolz-ms** [comment]
@lkupkepear thank you for bringing this problem to our attention. I can reproduce.

**Comment by karolz-ms** [comment]
For whatever reason the `v2/repositories/namespaces` query issued by VS Code only returns namespaces associated with the Docker Hub login. The same query used by the Docker Hub site UI returns both namespaces associated with the login, as well as namespaces owned by all organizations that the user belongs to.

Following up with Docker to figure out why the discrepancy...

**Comment by karolz-ms** [comment]
@nebuk89 @gtardif any idea why `v2/repositories/namespaces` query would skip organizational namespaces? LMK if there is a better repository to raise this question/open an issue--thanks!

**Comment by bwateratmsft** [comment]
Looks like we'll need to call `/v2/user/orgs` and dedupe orgs from what shows in namespaces. From my testing, orgs _do_ show in the `/v2/repositories/namespaces` query, but only if the requesting user is an owner in that org.

**Comment by bwateratmsft** [comment]
A fix for this has been released in Docker extension version 1.16.0.
